internal/quickstart/messages.go

```diff
@@ -73,16 +102,21 @@ func createFlag(client flags.Client, accessToken, baseUri, flagName, flagKey, pr
 			projKey,
 		)
 		if err != nil {
-			var e struct {
-				Code    string `json:"code"`
-				Message string `json:"message"`
-			}
-			_ = json.Unmarshal([]byte(err.Error()), &e)
-			existingFlag = e.Code == "conflict"
-			if !existingFlag {
-				return errMsg{err: errors.NewError(fmt.Sprintf("Error creating flag: %s. Press \"ctrl + c\" to quit.", e.Message))}
+			msgRequestErr, err := newMsgRequestError(err.Error())
```

Comment:
We have to handle this here since we don't consider a conflict an error, unlike the toggle flag command where a conflict is still an error.